### PR TITLE
Validator

### PR DESCRIPTION
**Pull request by liorblob**

### CODE REVIEW

File: demisto_sdk/validation/file_validator.py [hunk 1]
```diff
@@ -0,0 +1,450 @@
...
+        self._is_valid = True
+        self.configuration = configuration
+
+        if self.configuration.validate_conf_json:
+            self.conf_json_validator = ConfJsonValidator()
+        if self.configuration.validate_id_set:
+            self.id_set_validator = IDSetValidator(configuration=self.configuration)
+
+    @staticmethod
+    def is_py_script_or_integration(file_path):
```
Comment (Itay4) [inline demisto_sdk/validation/file_validator.py]: should be private?

File: demisto_sdk/validation/file_validator.py [hunk 2]
```diff
@@ -0,0 +1,450 @@
+#!/usr/bin/env python
```
Comment (Itay4) [inline demisto_sdk/validation/file_validator.py]: still need it now everything is py3?

File: demisto_sdk/validation/file_validator.py [hunk 3]
```diff
@@ -0,0 +1,450 @@
...
+            modified_files = modified_files.union(
+                modified_files_from_tag.intersection(non_committed_modified_files))
+
+            added_files = added_files.union(
+                added_files_from_tag.intersection(non_committed_added_files))
+
+            modified_files = modified_files - set(non_committed_deleted_files)
+            added_files = added_files - set(non_committed_modified_files) - set(non_committed_deleted_files)
+
+            # new_added_files = set([])
```
Comment (Itay4) [inline demisto_sdk/validation/file_validator.py]: ?

File: demisto_sdk/yaml_tools/unifier.py [hunk 4]
```diff
@@ -6,9 +6,12 @@
 import base64
 import re
 
+from demisto_sdk.common.tools import get_yaml
+from demisto_sdk.common.constants import TYPE_TO_EXTENSION
+
 
 class Unifier:
-    def __init__(self, package_path: str, dir_name: str, dest_path='', integration_prefix='integration',
+    def __init__(self, package_path: str, dir_name='Integrations', dest_path='', integration_prefix='integration',
```
Comment (Itay4) [inline demisto_sdk/yaml_tools/unifier.py]: why not using consts?

File: demisto_sdk/yaml_tools/unifier.py [hunk 5]
```diff
@@ -184,6 +187,22 @@ def insert_script_to_yml(self, script_type, yml_text, yml_data):
 
         return yml_text, script_path
 
+    def get_script_package_data(self):
```
Comment (Itay4) [inline demisto_sdk/yaml_tools/unifier.py]: test?

File: tests/conf_test.py [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+    {
+        "playbookID": "siri"
+    },
+    {
+        "playbookID": "alexa"
+    }
+]
+
+
+@pytest.mark.skip(reason="pending conf json fix")
```
Comment (Itay4) [inline tests/conf_test.py]: please add TODO to remove all the skips

File: tests/test_utils_test.py [hunk 7]
```diff
@@ -0,0 +1,30 @@
+from demisto_sdk.common import tools
```
Comment (Itay4) [inline tests/test_utils_test.py]: lets rename `utils_test` to `utils`

File: demisto_sdk/yaml_tools/unifier.py (hunk 5 above)
Comment (liorblob) [reply demisto_sdk/yaml_tools/unifier.py]: @guyfreund :)

**Pull request merged**